docs/content/preview/contribute/core-database/clion-setup.md

```diff
@@ -51,3 +51,23 @@ YB_USE_NINJA=0 ./yb_build.sh
 ```
 
 Subsequent builds can be launched also from CLion.
+
+### Configure as a compilation database project
+
+To improve performance, you can try opening the project as a compilation database project:
+
+1. Run `./yb_build.sh compilecmds` to generate the `compile_commands.json` file in the `build` directory.
+
+1. Move the `compile_commands.json` file to the project root.
```

Comment:
What is "opening the project" and "project root"?  Not sure if it is too obvious to state or whether it should be clarified.

Related to above

> Click **File > Open…** to open the project root directory.

Reply:
I assumed they were CLion specific and therefore known in context. Perhaps @kai-franz can clarify.

Reply:
Let's use "`yugabyte-db` folder" instead of "project root" to be more precise. Also, as I suggested in another comment, let's say to symlink it instead of moving the file.